### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by quant-ru** [kevinzg/facebook-scraper#162]
```py
for post in get_posts(names[0], pages=1):
    print(post['text'][:50])
``` 
error: 
``` HTTPError: 404 Client Error: Not Found for url: https://mobile.facebook.com/3624096454312871/posts/?_rdc=1&_rdr&refsrc=https%3A%2F%2Fm.facebook.com%2F3624096454312871%2Fposts%2F```

**Comment by balazssandor** [comment]
many pages don't have `/posts` path recently

**Comment by quant-ru** [comment]
Where is this file that creates a link to scrap?
 Its possible to just change it.

**Comment by GivenToFlyCoder** [comment]
Have the same error. HTTPError: 404 Client Error: Not Found for url: https://m.facebook.com/............................/posts/

**Comment by Mohamed01555** [comment]
how to solve this issue?

**Comment by Mohamed01555** [comment]
> Where is this file that creates a link to scrap?
> Its possible to just change it.

please tell me the file should be fixed

**Comment by quant-ru** [comment]
Yes, only thing I need is to change link creator. Many pages now don't have ```/posts``` so I just need to delete it

**Comment by Mohamed01555** [comment]
Now I have many files such that extractor.py, facebook_scrapper.py .. etc,  which of these files  should I modify link creator?

**Comment by kevinzg** [comment]
Should be fixed with #170.
Try with latest version.
